Utils/py_utils.py

```diff
@@ -18,3 +20,106 @@ def login_to_cli(self):
 def setup_cli(self):
     init_to_cli(self)
     login_to_cli(self)
+
+ # *************** VARIABLE ***************
+
+def set_variable(self, variable_id, value, exit_code=0):
+    return self.invoke_cli(self.cli_auth_params,
+                       ['variable', 'set', '-i', variable_id, '-v', value], exit_code=exit_code)
+
+def get_variable(self, *variable_ids):
+    return self.invoke_cli(self.cli_auth_params,
+                           ['variable', 'get', '-i', *variable_ids])
+
+def assert_set_and_get(self, variable_id):
+    expected_value = uuid.uuid4().hex
+
+    set_variable(self, variable_id, expected_value)
+    output = get_variable(self, variable_id)
+    self.assertEquals(expected_value, output.strip())
+
+def assert_variable_set_fails(self, variable_id, error_class, exit_code=0):
+    with self.assertRaises(error_class):
+        self.set_variable(variable_id, uuid.uuid4().hex, exit_code)
+
+def print_instead_of_raise_error(self, variable_id, error_message_regex):
+    output = self.invoke_cli(self.cli_auth_params,
+                             ['variable', 'set', '-i', variable_id, '-v', uuid.uuid4().hex], exit_code=1)
+
+    self.assertRegex(output, error_message_regex)
+ # *************** POLICY ***************
+
+def load_policy(self, policy_path):
+    return self.invoke_cli(self.cli_auth_params,
+                           ['policy', 'load', '-b', 'root', '-f', policy_path])
+
+def generate_policy_string(self):
+        variable_1 = 'simple/basic/{}'.format(uuid.uuid4().hex)
+        variable_2 = 'simple/space filled/{}'.format(uuid.uuid4().hex)
+        variable_3 = 'simple/special @#$%^&*(){{}}[]._+/{id}'.format(id=uuid.uuid4().hex)
+
+        policy = \
```

Comment:
This is unindented

Reply:
purposeful. Conjur policy shouldnt be indented because the """ is indentation sensitive. We want the conjur policy to start at the far left of the policy yaml not in the beginning. I will add a comment